src/thor/dao/release_dao.py

```diff
@@ -0,0 +1,179 @@
+import os
+import sqlalchemy as sa
+from . import config
+from models.models import Release
+
+from sqlalchemy.orm import sessionmaker
+from contextlib import contextmanager
+
+import logging
+
+from contextlib import contextmanager
+
+# Implements CRUD functions on the database.
+
+engine = sa.create_engine(config.RELEASE_DATABASE_URL)
+Session = sessionmaker(bind=engine)
+
+logging.basicConfig(level=os.environ.get("LOGLEVEL", "INFO"))
+logging.getLogger("sqlalchemy.engine").setLevel(logging.INFO)
+l = logging.getLogger(__name__)

```

Comment:
nit: Single character variable names aren't very readable, I'd call it `logger` or something.